src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -20,6 +20,13 @@
 
 namespace Microsoft.AspNetCore.SignalR.Client
 {
+    // TODO: Documentation
+    public enum HubConnectionStatus
+    {
+        Started,
```

Comment:
Hmm, would connected  make more sense here?

Reply:
Could do! I went with Started and Stopped because of StartAsync and StopAsync.

🚲 🏠 @davidfowl @anurse

Reply:
I like `Started` and `Stopped` as well, given the method names.

Reply:
Hmmm, the methods are Started and Stopped but they're on a class with connection in the name.

I think I prefer `Disconnected` and `Connected`.